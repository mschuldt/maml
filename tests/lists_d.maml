arduino = Arduino(desktop=True)

@block(once)
def fib():
    x = cons(10, None);
    x = cons(9, x)
    x = cons(6, cons(7, cons(8, x)))
    y = x
    while y:
        print_i(car(y))
        y = cdr(y)
    print_i(car(x))
    die(1)

arduino.send(fib)
