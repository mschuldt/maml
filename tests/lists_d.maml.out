Initializing avm...
Ready.

6
7
8
9
10
6
bye.
